index.html

```diff
@@ -0,0 +1,105 @@
+        <nav id="nav-bar">
+            <a class="nav-link" href="#premium">Features</a>
+            <a class="nav-link" href="#fast">How It Works</a>
+            <a class="nav-link" href="#quality">Pricing</a>
+        </nav>
+    </header> 
+    <main class="landingPage">
+        <h1> Handcrafted, home-made masterpieces</h1>
+
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input id="email" name="email" type="email" class="email" placeholder="Enter your email address" required>
+            <input id="submit" type="submit" class="submit">
+    </form> 
+</main>
+
+    <section>
+    <article id="premium">
+        <i id="fire" class="fa fa-3x fa-fire"></i>
+        <div class="article-bar">
+            
+            <h1>Premium Materials</h1>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </div>
+    </article>
+
+    <article id="fast">
+        <i id="truck" class="fa fa-3x fa-truck"></i>
+        <div  class="article-bar"> 
+            <h1>Fast Shipping</h1>
+            <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        </div>
+    </article>
+
+    <article id="quality">
+        <i id="battery" class="fa fa-3x fa-battery-full" aria-hidden="true"></i>
+        <div  class="article-bar">
+            <h1><a href="#pricing"></a>Quality Assurance</h1>
+            <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+        </div>
+    </article>
+
+</section>
+
+<iframe id="video" src="https://www.youtube.com/embed/W6NZfCO5SIk" allowfullscreen></iframe>
+
+<div class="articles">
+    <section class="side-band">
+        <h2>TENOR TROMBONE</h2> 
+        <strong>$600</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button type="button">SELECT</button>
+    </section>
+
+    <section class="side-band">
+        <h2>BASS TROMBONE</h2>
+        <strong>$900</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button type="button">SELECT</button>
+    </section>
+
+    <section class="side-band">
+        <h2>VALVE TROMBONE</h2>
+        <strong>$1200</strong>
+        <p>Lorem ipsum.Lorem ipsum.Lorem ipsum dolor.Lorem ipsum.</p>
+        <button type="button">SELECT</button>
+    </section>
+</div>
+
+
+
+<footer>
+    <div class="nav-end">
+        <a href="#">Privaty</a>
+        <a href="#">Terms</a>
+        <a href="#">Contact</a>
+    </div>
+    <p>Copyright 2016, Original Trombones</p>
+    
+</footer>
+    
+
+
+
+
+
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
**Hi VTY1999**
Great work so far, but you need to remove the script tag.

Reply:
Hi @VTY1999, you have done a great job so far reviewing your code. 
> This project is about the you performing a code review on this project by checking if you applied HTML & CSS best practices

I kindly suggest that you look through all of your code and find anywhere you might have not passed the standards, or a part of code you would refactor better 😄.

Start by "quoting the code", then state the aimed problem ❗ , politely offer a suggestion 😏 and conclude with a reason or importance of making that change.

Please follow the above guideline, ☝🏻 and make the review comment you made better! Also don't not forget to add as many reviews to your code as you can (the minimum is 2).